cmd/image.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// ImageCmd represents the image command
```

Comment:
Comment doesn't match name of function. Probably can just delete it. Same for other files.